recipes/pyqt_distutils/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyqt_distutils" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyqt-distutils-{{ version }}.tar.gz
+  sha256: ee5b8c30d3b0a473a5c23e43b2a29ce8e47fa7b48d8dc91816953da37bc053e8
+
+build:
+  entry_points:
+    - pyuicfg = pyqt_distutils.pyuicfg:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - docopt
+    - python >=3.6
+
+test:
+  imports:
+    - pyqt_distutils
+  commands:
+    - pip list
+    - pip check
+    - pyuicfg --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ColinDuquesnoy/pyqt_distutils
+  summary: A set of distutils extension for building Qt ui files
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - goanpeca
```

Comment:
Maintainer has not been active on the project for 5 years, so just me for now :)